.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2 # use CircleCI 2.0
+
+x-dockerbuild-7: &dockerbuild-7
+  steps:
+    - checkout
+    - run: sudo docker-php-ext-install sockets
+    - run: find . -name '*.php' -exec php -l {} > /dev/null \; 2> /tmp/lint.err ; test ! -s /tmp/lint.err
```

Comment:
Tiny nits that probably don't need fixing:
- `> /dev/null \; 2> /tmp/lint.err` would look more standard as `1>/dev/null \; 2>/tmp/lint.err`
- I'm not quite sure what the `\;` in this context is supposed to do.
- I think in terms of UX, it would be better to not overwrite the error file but to keep adding all problems to it and then display it if the test fails.

Reply:
I'll give this a go. TBH i just copy/pasted this from the travis job. 
Also +1 to squash 😄